### PR TITLE
Create shared pull request template

### PR DESCRIPTION
**Pull request by leebs778**
Following https://app.asana.com/0/1201760991851092/1202604754592955/f this should a standard readme format for repos

This is using github's recommendations https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#about-default-community-health-files -- note thata github requirement for shared templates is this `.github` repo being public so I've ensured the code governance doc is properly restricted to haus employees

### CODE REVIEW

**Comment by drorasaf** [comment]
Should we add the versioning question here?

**Comment by leebs778** [comment]
added versioning as an optional check

**Comment by leebs778** [comment]
ping here @jsonreeder @dubchuck

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Ticket
+
+## Goal
+
+## UX Changes (if any)
```
Comment (jsonreeder) [inline pull_request_template.md:5]: nit: no need to add "(if any)." All sections have that implication.

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Code Changes
+
+## Related Pull Requests
+
+## Notes & Questions for Reviewers
+
+### Checks
+- [ ] Code in this PR follows Haus' [code contribution guidelines](https://docs.google.com/document/d/1-3pOkN0Qk5OrsoDJCFPL_Jwunc6ptXLzSpTdTRAH7to/edit#heading=h.fy8j7l7xqyx8) or if it breaks guidelines I've explained why it's necessary.
+- [ ] \(Optional) I'm making a vmajor or vminor [Semver](https://semver.org/) change and have that present in the branch name
```
Comment (jsonreeder) [inline pull_request_template.md:15]: nit: As above, no need to add "optional" since all contents of this template are optional

**Review by jsonreeder — APPROVED** [review]
Great idea to add this cross repo. Thanks for the ping.

**Pull request merged**